src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -204,4 +204,174 @@ public int draw() {
         }
         return normalizedHistogram;
     }
-}
+
+    /**
+     * Determines if a variant would be called under the particular conditions of a given total depth, alt depth,
+     * average base qualities, allele fraction of variant and log odds threshold necessary to exceed to call variant.
+     * @param totalDepth Depth at the site to be called, both alt and ref.
+     * @param altDepth Number of alt bases at this site.
+     * @param averageQuality Average Phred-scaled quality of bases
+     * @param alleleFraction Allele fraction we are attempting to detect
+     * @param logOddsThreshold Log odds threshold necessary to exceed for variant to be called
+     * @return
+     */
+    public static boolean isCalled(int totalDepth, int altDepth, double averageQuality, double alleleFraction, double logOddsThreshold) {
```

Comment:
What, in words, is the criterion for `isCalled`?  I'm asking because `allele fraction` here seems possibly different from the command line argument allele fraction.  The latter, I think, is the true allele fraction, which the former can't be because only the alt and total depths, which are subject to sampling error, are observed.

I could imagine various ways to define theoretical sensitivity with the allele fraction in this method being:

* an exact value.  `isCalled` is true if the likelihood ratio between a) het with exactly this allele fraction and b) hom ref  is sufficiently large.
* a minimum. `isCalled` is true if the likelihood of being a het with *at least* this allele fraction is sufficiently large.
* irrelevant. `isCalled` could use a criterion that marginalizes out the latent true allele fraction and does model comparison between a) hom ref and b) int_0^1 likelihood(af) P(af) d af.  This is what Mutect now does.
* irrelevant. `isCalled` plugs in af = alt / total in calculating the likelihood.  This is what Mutect used to do.

I'm not saying any one is more right than the others, (well, I guess that between the last two my preference is clear) but I want to know which one the code implements.